Formula/python3.rb

```diff
@@ -33,15 +32,20 @@ class Python3 < Formula
   skip_clean "bin/easy_install3", "bin/easy_install-3.4", "bin/easy_install-3.5", "bin/easy_install-3.6"
 
   resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/87/1a/33d3d05569e857c5c5cc3e90d197bf4d9696dc740a05f66a09599d66e5bd/setuptools-32.2.0.zip"
-    sha256 "634313924fd186a2be0489c96965f5a909b666bd652eb3e16724913c707ec33f"
+    url "https://files.pythonhosted.org/packages/b5/e2/cca325e160b8e12334f5b2923ffcdb3c02da9fded1d2899d7930ac1c7448/setuptools-34.3.2.zip"
```

Comment:
We'll see what CI says but I'm pretty sure we have to change our setuptools/pip bootstrapping strategy to use ensurepip if we upgrade beyond 32.2; I can work on this later if you want to leave setuptools alone for now